scout4neo/scout4neo.go

```diff
@@ -0,0 +1,49 @@
+package scout4neo
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/project-regista/scout/configuration"
+
+	bolt "github.com/johnnadratowski/golang-neo4j-bolt-driver"
+)
+
+func StoreData(statement []string) {
+
+	configuration.LoadConfig()
+
+	a := configuration.DbAuth()
+	a.GetDB()
+
+	driver := bolt.NewDriver()
+	conn, err := driver.OpenNeo(a.URL)
+
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	defer conn.Close()
+
+	for i := 0; i < len(statement); i++ {
```

Comment:
This is absolutely correct. But just so you know. The idiomatic way to write it in Go is:

`for _, v := range statement {...}` 

where `v` would be the value at the index `i` -> `value[i]`
since we dont need `i` we `_`